Gopkg.toml

```diff
@@ -53,6 +53,13 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.12.6"
 
+# manifestival is a testing only dep.
+[[constraint]]
+  name = "github.com/jcrossley3/manifestival"
+  # HEAD as of 2019-08-01
+  revision = "dbd2aec45641ad6a8f9834242f6ae5616cdb7bb1"
+  source = "github.com/n3wscott/manifestival"
```

Comment:
Plans to unfork?

Reply:
no. not until they stop using controller runtime client.